popcoder/video.py

```diff
@@ -200,7 +205,7 @@ def preprocess(self, data):
         print 'Beginning pre-process...'
         for url, video in data['media'][0]['clipData'].iteritems():
             print 'Downloading {0} from {1}.'.format(video['title'], url)
-            video['title'] = video['title'].replace(' ', '-') + '.webm'
+            re.sub(r':(?=..(?<!\d:\d\d))|[^a-zA-Z0-9 ](?<!\.)', '', video['title'])
```

Comment:
Uh? What does that do? 
A comment would be nice :)
Anyway, this won't *change* `video['title']` since its immutable. So this line doesn't change anything.